### PR TITLE
Enhancement: implements development good practices to give form shortcode

### PR DESCRIPTION
**Pull request by glaubersilva**
Resolves [GIVE-561]

## Description

This PR implements sanitizing and escaping good practices intended to improve code quality.

## Affects

Shortcodes.

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [ ] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

[GIVE-561]: https://stellarwp.atlassian.net/browse/GIVE-561?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Views/IframeView.php [hunk 1]
```diff
@@ -273,8 +278,8 @@ class="js-give-embed-form-modal-opener"
 				data-form-id="%1$s"%3$s>%2$s</button>
 			</div>',
             $this->uniqueId,
-            $this->buttonTitle,
-            $this->buttonColor ? " style=\"background-color: {$this->buttonColor}\"" : ''
+            $buttonTitle,
+            $this->buttonColor ? " style=\"background-color: {$buttonColor}\"" : ''
```
Comment (jonwaldstein) [inline src/Views/IframeView.php:282]: Can we actually use late escaping here? That way the previous variables can not be altered and it is more clear when looking at the sprintf

**Review by jonwaldstein — CHANGES_REQUESTED** [review]

File: src/Views/IframeView.php (hunk 1 above)
Comment (glaubersilva) [reply src/Views/IframeView.php]: Resolved in 1b785dd4e74939d345c573eec409f13e1420e736

**Comment by glaubersilva** [comment]
@jonwaldstein This is ready for re-review.

**Review by jonwaldstein — APPROVED** [review]
@glaubersilva thanks for the update, looks good 👍

**Pull request merged**